From e762fa94f4473974e7e8acabb778d4a75b4e10d2 Mon Sep 17 00:00:00 2001
From: Craig Kewley <craigx.kewley@intel.com>
Date: Thu, 4 Aug 2016 16:54:01 +0100
Subject: [PATCH 045/102] ASoC: Intel: Skylake: enable i915 code.

Enabl i915_init code by removing '#if 0' as it is used on the GP MRB board.

Signed-off-by: Craig Kewley <craigx.kewley@intel.com>

Conflicts:
	sound/soc/intel/skylake/skl.c
---
 sound/soc/intel/skylake/skl.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sound/soc/intel/skylake/skl.c b/sound/soc/intel/skylake/skl.c
index d8f92f6..a5c833e 100644
--- a/sound/soc/intel/skylake/skl.c
+++ b/sound/soc/intel/skylake/skl.c
@@ -681,6 +681,7 @@ static void __maybe_unused skl_codec_create(struct hdac_ext_bus *ebus)
 	.get_response = snd_hdac_bus_get_response,
 };
 
+#if 1
 static int skl_i915_init(struct hdac_bus *bus)
 {
 	int err;
@@ -699,6 +700,7 @@ static int skl_i915_init(struct hdac_bus *bus)
 
 	return err;
 }
+#endif
 
 static void skl_probe_work(struct work_struct *work)
 {
-- 
1.9.1

